docs/usage/slaveloan/index.rst

```diff
@@ -0,0 +1,19 @@
+Loaning Slaves
+==============
+
+The ``/slaveloan`` API endpoint.
```

Comment:
A little bit more narrative would be nice here, to familiarize readers with the kinds of objects used in the interface.